lxd/db/cluster/schema.go

```diff
@@ -84,7 +84,7 @@ CREATE TABLE images_source (
 );
 CREATE TABLE "instances" (
     id INTEGER primary key AUTOINCREMENT NOT NULL,
-    node_id INTEGER NOT NULL,
+    node_id INTEGER,
```

Comment:
An instance doesn't have this issue, it always has a single node that it's stored on, so no need to change that table.